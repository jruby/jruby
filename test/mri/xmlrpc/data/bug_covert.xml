<?xml version="1.0"
encoding="ISO-8859-1"?><methodResponse><params><param><value>Site,SANs,Array
Configured Capacity,Array Reserved Capacity,Array Ava
ilable Capacity,Array % Reserved,Host Allocated,Host Used,Host Free,Host %
Used
</value></param></params></methodResponse>
