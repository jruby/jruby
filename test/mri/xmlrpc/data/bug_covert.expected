--- 
- true
- >
  Site,SANs,Array

  Configured Capacity,Array Reserved Capacity,Array Ava

  ilable Capacity,Array % Reserved,Host Allocated,Host Used,Host Free,Host %

  Used
